src/PagesComponents/PersonLink.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  selectedPersonSlug: string | undefined;
+};
+
+export const PersonLink: React.FC<Props> = ({ person, selectedPersonSlug }) => {
+  const isSelectedPerson = selectedPersonSlug === person.slug;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames(
+        { 'has-background-warning': isSelectedPerson },
+        { '': !isSelectedPerson },
```

Comment:
```suggestion
```
This is completely unnecessary, if previous class won't be added `classNames` just returns empty sring by itself.